components/SSI/AddFunds/index.tsx

```diff
@@ -683,20 +719,6 @@ function Component(props: InputType) {
                 )} */}
                 {
                   <>
-                    <h3 style={{ marginTop: "7%" }}>
-                      Add funds into{" "}
-                      {type === "buy" ? (
```

Comment:
not sure this should be removed @DevPH-Wistkey

Reply:
Hi @tralkan I'm referring to Sherly comment here: https://www.notion.so/ssiprotocol/s8-issue-Sherly-52dc2afd7bdd43159a68f20dee8ed888#d3abb4da64e4427cbe0757cad1ff3ce9

Reply:
I see but I'll find another way to summarise.

Reply:
Should I put back the deleted component? @tralkan

Reply:
yes please @DevPH-Wistkey

Reply:
Solved